src/mandrel.ts

```diff
@@ -0,0 +1,57 @@
+import * as c from './constants'
+import {downloadAndExtractJDK, getLatestRelease} from './utils'
+
+const MANDREL_REPO = 'mandrel'
+const MANDREL_TAG_PREFIX = 'mandrel-'
```

Comment:
Why not `MANDREL_NAMESPACE` from `./constants`?

Reply:
`MANDREL_NAMESPACE` is specified by this GHA and `MANDREL_TAG_PREFIX` is an external dependency. They happen to be the same right now, but that could potentially change in the future. Maybe `MANDREL_NAMESPACE` could be `MANDREL_TAG_PREFIX`...I'll have a look if that works.